### PR TITLE
Add actual onboarding slides

### LINKED ISSUE #7012: Add actual onboarding for Android
This is about adding the actual onboarding steps to the onboarding screens.

Bear in mind that this will be the first screen the user will see when first opening the app, so take extra care here, everything has to be perfect, beautiful, animate when possible, load fast and generally be a pleasure to use.

We should also add analytics on what steps does the user view and more.

#6998 should be done first.

Designs and full specs: https://www.notion.so/toggl/Design-specs-for-onboarding-carousel-a55437219e074f6a869ab4f9ab8e5ff4

**NOTE:** This issue is part of the `Onboarding refactor` project and it should be merged into the project branch and not into develop.

### PR DESCRIPTION
**Pull request by pdrj**
## What's this?
This adds the onboarding slides.

### Relationships

Closes #7012 

## Why do we want this?
It's a part of the project. 

## How is it done?
Using viewpager and stuff.

### Why not another way?
I don't like custom gesture handling.

## Review hints
This is incomplete.

We're missing:
- assets, so that it looks fine on smaller screens too
- looping viewpager (after the last slide, go to the first one) - all Android approaches I know are hacky and I don't really like those
- testing on slow devices (especially lottie and the periscope)
- propably something else I forgot.

I'm putting it out there so interested people can already take a look.

## :squid: Permissions
Nope. 🛑

### CODE REVIEW

File: Toggl.Droid/Toggl.Droid.csproj [hunk 1]
```diff
@@ -114,6 +114,10 @@
       <HintPath>..\packages\Accord.Statistics.3.8.0\lib\netstandard2.0\Accord.Statistics.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Lottie.Android, Version=3.4.0.0, Culture=neutral, PublicKeyToken=null">
+      <HintPath>..\packages\Com.Airbnb.Android.Lottie.3.4.0\lib\monoandroid10.0\Lottie.Android.dll</HintPath>
+      <Private>True</Private>
```
Comment (daividssilverio) [inline Toggl.Droid/Toggl.Droid.csproj]: 👀

File: Toggl.Droid/Views/SurfaceViewWithBackground.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+using Android.Graphics;
+using Android.Runtime;
+using Android.Text;
+using Android.Util;
+using Android.Views;
+
+namespace Toggl.Droid.Views
+{
+    [Register("toggl.droid.views.SurfaceViewWithBackground")]
+    public class SurfaceViewWithBackground : SurfaceView
```
Comment (daividssilverio) [inline Toggl.Droid/Views/SurfaceViewWithBackground.cs]: This isn't being used, is it?

File: Toggl.Shared/Resources.resx [hunk 3]
```diff
@@ -138,9 +138,19 @@ Let’s get you set up to start tracking.</value>
         <comment>Displayed in the login view right above the email text field</comment>
     </data>
     <data name="OnboardingMessageVariantA" xml:space="preserve">
-        <value>Simple time tracking.
-Powerful reporting.</value>
-        <comment>Message shown in the onboarding view</comment>
+        <value>Time tracking so simple
```
Comment (daividssilverio) [inline Toggl.Shared/Resources.resx]: Wait, shouldn't this be the same as iOS? Are our copies different?

Comment (pdrj) [reply Toggl.Shared/Resources.resx]: I don't know - I took it from the notion page: https://www.notion.so/toggl/Design-specs-for-onboarding-carousel-a55437219e074f6a869ab4f9ab8e5ff4 - I'll ask in the channel.

Comment (pdrj) [reply Toggl.Shared/Resources.resx]: This is the correct one it turns out :)

File: Toggl.Droid/Views/SurfaceViewWithBackground.cs (hunk 2 above)
Comment (pdrj) [reply Toggl.Droid/Views/SurfaceViewWithBackground.cs]: Nope, sorry.

**Comment by pdrj** [comment]
I've made the requested changes - I guess we'll still hold off on this one till tomorrow, since we were unable to get the lottie files properly exported today.

**Pull request merged**